domains/POAS/events/lpa-access-granted/index.md

```diff
@@ -0,0 +1,31 @@
+---
+name: lpa-updated
+version: 0.0.1
+summary: |
+  The LPA's details have changed
+producers:
+  - opg.poas.lpastore
```

Comment:
MRLPA is the producer here as LPA store knows nothing of the onelogin ids associated with the application.